docs/sql-guide/functions/function-parsetimestamp.md

```diff
@@ -0,0 +1,138 @@
+---
+title: PARSETIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+## Before you begin
```

Comment:
Before you begin goes after the first heading and the introductory text.

Reply:
But this link is actually information that's better right at the end under "Further information" because it gives the user additional information they can look into once they've gone through the content.

Reply:
We dont need that and it was left by mistake. so removed in latest commit.